src/features/workspaces/workspace-detail/WorkspaceDetail.tsx

```diff
@@ -236,7 +238,22 @@ export const WorkspaceDetail = () => {
                       clearAllFilters={clearAllFilters}
                     />
                   ) : (
-                    <div className="pf-v5-u-background-color-100 pf-v5-u-p-lg pf-v5-u-text-align-center">Roles assigned outside - Coming soon</div>
+                    <ParentRoleAssignmentsTable
+                      groups={groups}
+                      totalCount={groupsTotalCount}
+                      isLoading={groupsIsLoading}
+                      page={page}
+                      perPage={perPage}
+                      onSetPage={onSetPage}
+                      onPerPageSelect={onPerPageSelect}
+                      sortBy={sortBy}
+                      direction={direction}
+                      onSort={onSort}
+                      filters={filters}
+                      onSetFilters={onSetFilters}
+                      clearAllFilters={clearAllFilters}
```

Comment:
We should keep these props in the `ParentRoleAssignmentsTable` same for the `RoleAssignmentsTable`. These are state props of the table we don't want to share them between these 2 tables.

Reply:
If we're going to reuse `RoleAssignmentsTable` we should have two separate state values for each table.